app/allocations/middleware.test.js

```diff
@@ -35,7 +35,7 @@ describe('#setAllocationsSummary', function() {
     expect(locals.allocationsSummary).to.deep.equal([
       {
         active: false,
-        label: 'allocations::dashboard.label',
+        label: 'allocations::dashboard.labels.single_requests',
```

Comment:
Has this been squashed into the wrong commit? There's no update to the middleware as part of this change?

Reply:
Sorry, yes, this belongs to another commit. Shall amend.